packages/foreman/rubygem-netrc/rubygem-netrc.spec

```diff
@@ -71,6 +73,9 @@ popd
 %{gem_instdir}/test
 
 %changelog
+* Mon Apr 29 2019 Evgeni Golov - 0.11.0-4
+- rebuilt
```

Comment:
I'm sure there's a better description here :)

Reply:
Picky.

Reply:
You know me :)